### DISCUSSION TITLE
绳网什么时候开检举功能？

### DISCUSSION THREAD

**Discussion by bin-lin2** [share121/inter-knot#1636] [其他]
真是绳网大了什么匠都有
![微信图片_20240812130641](https://github.com/user-attachments/assets/06552ea5-e750-4afe-9359-d0228fb21e9b)
有机器人脚本回复的
有发学习版游戏的
还有到各个帖子看都不看就打广告的
真不想让这写些东西污染绳网啊

**Comment by bin-lin2** [comment]
当然，我指的不是玩梗广告的那行

**Comment by capacitor1** [comment]
学习版游戏我倒觉得没什么，反倒如果刚好想玩，这还有帮助。

**不过广告这东西我也反对，如果是盈利目的，那必须正义制裁！**

**Comment by aiTunes-dev** [comment]
**已经见到管理员标签用户了，估计很快就有了**

**Reply by aiTunes-dev** [reply]
<a href="https://github.com/share121/inter-knot/discussions/1378">#1378</a>
